File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        sort(salary.begin(),salary.end());
        //int low = salary[0],high =salary[salary.size()-1];
        int size = salary.size();
        long double  sum = 0;
        for(int i = 1 ; i <size-1;i++)
            sum += salary[i];
        return (sum/(size-2));
        
    }
};

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.py
class Solution:
    def average(self, salary: List[int]) -> float:
        temp = sorted(salary)
        min_,max_= temp[0],temp[-1]
        len_  = len(temp)
        sum_ = sum(temp) - min_-max_
        return (sum_)/(len_-2)
        